bcgov/invasivesbc
# redux record sets

- Add record set to store with add functionality [rebase]
- Set up default record sets and proper activity pulling from store on activities page [rebase]
- Refactor remove record set functions to redux saga [rebase]
- Reduxify selected record [rebase]
- Reduxify adding boundary to record set [rebase]
- Reduxify removal of boundary [rebase]
- Remove props.children from recordSetContext
- Remove provider and prep for setting record set WIP [rebase]
- Remove reliance on recordSetContext and delete file
- Fix boundary chip and clean related console log errors
- Fix record set expand functionailty
- Set up set record set which fixes other recordSetAccordian functions [rebase]
- Handle the console log errors for uncontrolled name input
- Fix name autofill and put bandaid over delete dialog typography dom warning
- Rid console log warning of boundary select and have default show up properly
- Fix default sets, fix add record index, and remove localstorage getters in saga

# Overview

This PR includes the following proposed change(s):

- {List all the changes, if possible add the linked issue/ticket #}
- reduxify record sets 
- #2130
- removed recordSetContext
- fixed various issues including adding record set indexing and console log warnings
- store boundaries in userSettings state
- store records sets in userSettings state
- store selected record in userSettings state

## Type of change

- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- Manually tested by
- 	creating records
-	creating boundaries
-	testing to see if they return what's expected
-	creating record sets
-	deleting record sets in and out of order
-	editing a record set name, colours, etc...
-	selecting various records

## Screenshots

Please add any relevant UI screenshots if applicable.

## Checklist:
